cli/connhelper/commandconn/commandconn_unix_test.go

```diff
@@ -4,12 +4,17 @@ package commandconn
 
 import (
 	"context"
+	"errors"
 	"io"
 	"io/fs"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strconv"
 	"testing"
 	"time"
 
-	"github.com/docker/docker/pkg/process"
+	"golang.org/x/sys/unix"
```

Comment:
Actually, can use `syscall` for this; if we want to move the connhelper to  a separate module, we'd be able to reduce dependencies.